### PR TITLE
Wave2 1011 help text tweaks

### PR DESCRIPTION
**Pull request by nindusaini**

### CODE REVIEW

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/templates/node--gbl-supervisor.tpl.php [hunk 1]
```diff
@@ -37,8 +37,8 @@
         // Create a variable with the image style url
         $bg_img = file_create_url(image_style_url('global_supervisor_img', $content['field_gbl_supervisor_image']['#items'][0]['uri']));
 
-        // And add the alt tag.
-        $alt = isset($content['field_gbl_supervisor_image']['#items'][0]['alt']) ? $content['field_gbl_supervisor_image']['#items'][0]['alt'] : NULL;
+        // set the alt to be supervisor's full name
+        $alt = $fname . ' ' . $lname;
```
Comment (balrajb) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/templates/node--gbl-supervisor.tpl.php]: we should check $fname and $lname exist!
actually having bit more explanatory variables names would help next develop.

**Comment by nindusaini** [comment]
Hi Balraj,
Thanks for the review, here's the explanation to your questions:
Don't see alt for Profile Picture / image : we decided to not to have alt field, we're using supervisor's name as alt in template.
 "Provide a brief explanation of the type of candidate you're looking for" should be help text for "More information" field or label? : Double checked with Jaela, this is the way we want it with no help text.
    not a part of this PR but "// Define the faculty (no if statment needed because it's a required field)
    " should be // Define the faculty (no if statement needed because it's a required field)  will fix the typo

**Comment by balrajb** [comment]
@nindusaini thanks for confirming items were contradictory with AC, approved!

File: sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/templates/node--gbl-supervisor.tpl.php (hunk 1 above)
Comment (nindusaini) [inline sites/all/modules/ucalgary/ucgy_gbl/ucgy_gbl_supervisor/templates/node--gbl-supervisor.tpl.php]: Variables changed to first_name and last_name, we do not need test they exist as they are required fields.

**Comment by dpfitzsi** [comment]
Balraj can resolve this conflict?

**Comment by dpfitzsi** [comment]
Manually resolved conflict.

**Pull request merged**